Description: Configuration adjustment
 This patch adjusts include/polarssl/config.h to some Debian specific
 capabilities
Author: Arnaud Cornet <arnaud.cornet@gmail.com>

--- polarssl-1.2.8.orig/include/polarssl/config.h
+++ polarssl-1.2.8/include/polarssl/config.h
@@ -627,8 +627,8 @@
  * Requires: POLARSSL_TIMING_C
  *
  * Uncomment to enable the HAVEGE random generator.
-#define POLARSSL_HAVEGE_C
  */
+#define POLARSSL_HAVEGE_C
 
 /**
  * \def POLARSSL_MD_C
@@ -652,8 +652,8 @@
  *
  * Uncomment to enable support for (rare) MD2-signed X.509 certs.
  *
-#define POLARSSL_MD2_C
  */
+#define POLARSSL_MD2_C
 
 /**
  * \def POLARSSL_MD4_C
@@ -665,8 +665,8 @@
  *
  * Uncomment to enable support for (rare) MD4-signed X.509 certs.
  *
-#define POLARSSL_MD4_C
  */
+#define POLARSSL_MD4_C
 
 /**
  * \def POLARSSL_MD5_C
